.github/workflows/conformance.yml

```diff
@@ -4,8 +4,7 @@ on:
   push:
     branches:
       - main
-  pull_request_target:
-    types: [labeled]
+  pull_request:
```

Comment:
If this is not a particularly expensive operation, I would like the config to match our other CI?
```
on:
  push:
    branches:
      - '*'
  pull_request:
    branches:
      - main
```

Reply:
yeah that would be great.

Reply:
Done! I made the pattern `**`, since a single `*` will only match branches that don't contain a `/`.